MANIFEST.in

```diff
@@ -1,6 +1,7 @@
 include README.md LICENSE settings.yml
 recursive-include buildtest *
 graft buildtest
+include settings.yml
```

Comment:
Trying to understand why we need `settings.yml` in MANIFEST.in. If i understand the first line also contains `settings.yml` and that should be removed. The `graft buildtest` in line 3 should take care of all files under `buildtest` directory. See https://docs.python.org/2/distutils/sourcedist.html#commands or https://packaging.python.org/guides/using-manifest-in/#manifest-in-commands

Reply:
Oh that shouldn't have been added, I was playing around with settings and forgot to remove it.
Totally agree on utils/commands.py, I can do that in another PR.